plugins/providers/gcloudiam/client.go

```diff
@@ -25,11 +24,7 @@ type iamClient struct {
 
 func newIamClient(credentialsJSON []byte, resourceName string) (*iamClient, error) {
 	ctx := context.Background()
-	opts := []option.ClientOption{
-		option.WithCredentialsJSON(credentialsJSON),
-		option.WithHTTPClient(tracing.NewHttpClient("GCloudIAMHttpClient")),
-	}
-	cloudResourceManagerService, err := cloudresourcemanager.NewService(ctx, opts...)
+	cloudResourceManagerService, err := cloudresourcemanager.NewService(ctx, option.WithCredentialsJSON(credentialsJSON))
```

Comment:
need your help to revert on gcs provider as well
https://github.com/odpf/guardian/blob/ccaff0e506d4cac7eb62db0f7f0027012ef05980/plugins/providers/gcs/client.go#L27-L31